mcrouter/test/MCProcess.py

```diff
@@ -256,7 +255,7 @@ def ensure_connected(self):
             while True:
                 try:
                     res = self.thrift_client.mcVersion()
-                    if res == Result.OK:
+                    if res == carbon.carbon_result.thrift_types.Result.OK:
```

Comment:
While this could be generated from the Carbon IDLs, `createThriftTestClient` isn't implemented in the OSS version of `mcrouter_config.py`, so it seems better to "hide" the Meta-specific bits instead.